README.md

```diff
@@ -20,7 +20,7 @@ $ cd build && make test
 
 ## Example
 
-A easy demo showing how to create a binary tree.
+The following is an easy demo that shows how to create a binary tree.
```

Comment:
Thanks, nice suggestion.
I'd like to use `The following is an easy demo showing how to create a binary tree.` since i think it's a complete sentence.